src/DynamicBody.cpp

```diff
@@ -67,6 +67,11 @@ DynamicBody::DynamicBody(const Json &jsonObj, Space *space) :
 		throw SavedGameCorruptException();
 	}
 
+	// fix saves with nans
```

Comment:
I'd recommend this comment be annotated with the current savefile version (see Game.cpp) so that the workaround can be removed when the next savebump occurs.

Reply:
I think we used to add "todo" notes to SAVEBUMP.txt as well, things to fix for next release, so that might be a good idea.
I should add "checking SAVEBUMP.txt" to the release guide, if we think this is a good strategy?

Reply:
That would be a good idea on both counts; this definitely deserves to be in the SAVEBUMP.txt file as well.